moshi/src/main/java/com/squareup/moshi/JsonReader.java

```diff
@@ -514,8 +514,19 @@ public final void setFailOnUnknown(boolean failOnUnknown) {
   /**
    * Changes the reader to treat the next name as a string value. This is useful for map adapters so
    * that arbitrary type adapters can use {@link #nextString} to read a name value.
+   *
```

Comment:
suuuuuper nit: this is Java and you need to take a `<p>` 😢

Reply:
OMG how quickly I forget!